python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -870,14 +875,42 @@ def evaluate_tick(self, context: "SensorEvaluationContext") -> "SensorExecutionD
                     check.failed("Expected a single SkipReason: received multiple SkipReasons")
 
         _check_dynamic_partitions_requests(dynamic_partitions_requests)
-        resolved_run_requests = [
-            run_request.with_replaced_attrs(
-                tags=merge_dicts(run_request.tags, DagsterRun.tags_for_sensor(self)),
-            )
-            for run_request in self.resolve_run_requests(
-                run_requests, context, self._asset_selection, dynamic_partitions_requests
-            )
-        ]
+
+        run_requests_to_resolve = []
+        resolved_run_requests = []
+        for run_request in run_requests:
+            if run_request.requires_backfill_daemon:
```

Comment:
check pyright for this. might need to do a `isinstance(run_request, MultiRunRequest)` instead